doc/chapters/filesystem/index.rst

```diff
@@ -85,7 +85,7 @@ protection for the filesystem by only allowing a final write of the changes in
 the upper layer when the container is shutting down.
 
 To enable the write buffer, set the ``enableWriteBuffer`` option in the
-``com.pelagicore.SoftwareContainerAgent.CreateContainer(config)`` call.
+``com.pelagicore.SoftwareContainerAgent.Create(config)`` call.
 This is done using the config parameter in specific, for example, this JSON
 config::
```

Comment:
Here is write with a different mark-up than the new text. Maybe we should use this for the new chapter also.